tests/integration/components/osf-copyright/component-test.js

```diff
@@ -2,25 +2,13 @@ import {
     moduleForComponent,
     test
 } from 'ember-qunit';
-import hbs from 'htmlbars-inline-precompile';
+
```

Comment:
With the removal of this code, what exactly does this unit test do?

I'm glad you're cleaning up unused code, but if I read this correctly, we should either:
- Write a test that does something, or,
- Remove this file completely (because it makes the test suite run longer without providing any information)

Long term I lean towards option A, but I'm not sure when we'll start emphasizing tests. Ember-osf test coverage has historically been a bit low.